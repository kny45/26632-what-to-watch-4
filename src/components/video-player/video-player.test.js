import React from "react";
import renderer from "react-test-renderer";
import VideoPlayer from "./video-player.jsx";

describe(`Render VideoPlayer`, () => {
  const mock = {
    previewImage: `image`,
    previewVideoLink: `videoLink`
  };

  it(`VideoPlayer is rendered correctly with play`, () => {
    const tree = renderer.create(
        <VideoPlayer
          isPlaying = {true}
          movieCard = {mock}
        />, {
          createNodeMock: () => {
            return {};
          }
        }
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`VideoPlayer is rendered correctly with pause`, () => {
    const tree = renderer.create(
        <VideoPlayer
          isPlaying = {false}
          movieCard = {mock}
        />, {
          createNodeMock: () => {
            return {};
          }
        }
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
