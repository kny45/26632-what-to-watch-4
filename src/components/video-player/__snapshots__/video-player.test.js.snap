// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render VideoPlayer VideoPlayer is rendered correctly with pause 1`] = `
<video
  height="175"
  width="280"
/>
`;

exports[`Render VideoPlayer VideoPlayer is rendered correctly with play 1`] = `
<video
  height="175"
  width="280"
/>
`;
